docs/src/Groups/permgroup.md

```diff
@@ -76,11 +102,14 @@ parent(x)
 parent(y)
 ```
 
+
```

Comment:
Please try not to add spurious new lines for no apparent reason (it can easily happen! Just make sure to review the PR one last time before you submit it (or also *after* you submitted it, via the webview), and then directly address such minor issues.

Reply:
Should be all better now